e2e/startup-profiles.e2e.spec.ts

```diff
@@ -119,18 +135,22 @@ test.describe.serial('track startup windows based on existing profiles and setti
       skipReasons = await verifyNoSystemProfile();
     });
 
-    test('nonexistent settings', async() => {
+    test('nonexistent settings act like an empty default profile', async() => {
       test.skip(skipReasons.length > 0, `Profile requirements for this test: ${ skipReasons.join(', ') }`);
       if (process.platform === 'win32') {
-        await createNonexistentDataUserRegistryProfile();
+        await createDefaultUserRegistryProfileWithNonexistentFields();
       } else {
-        // Circumvent the type-checker by json-parsing a string of non-settings
-        const s = `{ "fruits": {"oranges": 5, "mangoes": true, "citrus": "lemons" } }`;
-        const s1 = JSON.parse(s);
+        const s1 = {
+          version: 10,
+          fruits:  {
+            oranges: 5, mangoes: true, citrus: 'lemons',
+          },
+        } as unknown as RecursivePartial<Settings>;
 
         await createUserProfile(s1 as RecursivePartial<Settings>, null);
```

Comment:
`s1` is already `RecursivePartial<Settings>`, there's no need for the cast here.